clinica/pipelines/t1_linear/t1_linear_pipeline.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+import clinica.pipelines.engine as cpe
+
+
```

Comment:
When I run your pipeline, I noticed that hases were not generated in working directory:
```text
$ pwd
/home/alexandre.routier/Temp/LightDataset/WD/t1-linear
$ ls
d3.js
graph1.json
graph.json
index.html
_t1w_..home..alexandre.routier..Temp..LightDataset..BIDS_T1_LINEAR..sub-ADNI003S4081..ses-M00..anat..sub-ADNI003S4081_ses-M00_T1w.nii
_t1w_..home..alexandre.routier..Temp..LightDataset..BIDS_T1_LINEAR..sub-CAPP01001TMM3B0..ses-M00..anat..sub-CAPP01001TMM3B0_ses-M00_T1w.nii.gz
_t1w_..home..alexandre.routier..Temp..LightDataset..BIDS_T1_LINEAR..sub-CAPP01011PTC..ses-M00..anat..sub-CAPP01011PTC_ses-M00_T1w.nii
```
When using iterables, folder names can be very long and possibly generate ` OSError`. In (almost?) all pipelines, we added the following lines:

```python
# Use hash instead of parameters for iterables folder names
# Otherwise path will be too long and generate OSError
from nipype import config
cfg = dict(execution={'parameterize_dirs': False})
config.update_config(cfg)
```

This has been added to PR #63 regarding Jinja2 hoping we won't face this issue again. :)